modules/claims_api/spec/sidekiq/poa_updater_spec.rb

```diff
@@ -186,6 +200,32 @@
     end
   end
 
+  describe 'when the claims_api_use_person_web_service flipper is on' do
```

Comment:
@tycol7  Do you see why this test isn't passing?

Reply:
There is a typo between [`claims_api_use_person_web_service`](https://github.com/department-of-veterans-affairs/vets-api/blob/8736614aa6b2996e2891ba448f49c828c6e5b944/modules/claims_api/spec/sidekiq/poa_updater_spec.rb#L208C50-L208C83) and [`claims_api_use_vet_record_service`](https://github.com/department-of-veterans-affairs/vets-api/blob/8736614aa6b2996e2891ba448f49c828c6e5b944/modules/claims_api/app/sidekiq/claims_api/poa_updater.rb#L78), resulting in us calling external BGS.

Reply:
Thanks